Test failure (res.toError() should return an Error)

When doing a `make test`, one of the tests is failing.  Here it is with Node 5.0.0:

```
  1) res.toError() should return an Error:
     Uncaught AssertionError: err.status == 400
      at /Users/chris/Downloads/superagent-latest/superagent/test/node/toError.js:20:7
      at Request.callback (/Users/chris/Downloads/superagent-latest/superagent/lib/node/index.js:785:12)
      at Stream.<anonymous> (/Users/chris/Downloads/superagent-latest/superagent/lib/node/index.js:993:12)
      at emitNone (events.js:72:20)
      at Stream.emit (events.js:166:7)
      at Unzip.<anonymous> (/Users/chris/Downloads/superagent-latest/superagent/lib/node/utils.js:108:12)
      at emitNone (events.js:72:20)
      at Unzip.emit (events.js:166:7)
      at endReadableNT (_stream_readable.js:905:12)
      at doNTCallback2 (node.js:450:9)
      at process._tickCallback (node.js:364:17)
```

Similarly, with Node 0.12.7:

```
  1) res.toError() should return an Error:
     Uncaught AssertionError: err.status == 400
      at /Users/chris/Downloads/superagent-latest/superagent/test/node/toError.js:20:7
      at Request.callback (/Users/chris/Downloads/superagent-latest/superagent/lib/node/index.js:785:12)
      at Stream.<anonymous> (/Users/chris/Downloads/superagent-latest/superagent/lib/node/index.js:993:12)
      at Stream.emit (events.js:129:20)
      at Unzip.<anonymous> (/Users/chris/Downloads/superagent-latest/superagent/lib/node/utils.js:108:12)
      at Unzip.emit (events.js:129:20)
      at _stream_readable.js:908:16
      at process._tickCallback (node.js:355:11)
```

--- a/test/node/toError.js
+++ b/test/node/toError.js
@@ -9,12 +9,12 @@ app.get('/', function(req, res){
   res.status(400).send('invalid json');
 });
 
-app.listen(8888);
+app.listen(65197);
 
 describe('res.toError()', function(){
   it('should return an Error', function(done){
     request
-    .get('http://localhost:8888/')
+    .get('http://localhost:65197/')
     .end(function(err, res){
       var err = res.toError();
       assert(err.status == 400);

